src/test/features/claimant-response/routes/part-payment-received.ts

```diff
@@ -0,0 +1,138 @@
+import { expect } from 'chai'
+import * as request from 'supertest'
+import * as config from 'config'
+
+import { attachDefaultHooks } from '../../../routes/hooks'
+import { app } from 'main/app'
+import * as claimStoreServiceMock from '../../../http-mocks/claim-store'
```

Comment:
please replace relative paths i.e `'../../../http-mocks/claim-store'` with `test/http-mocks/claim-store`. Please do for all instances.